packages/@aws-cdk-containers/ecs-service-extensions/lib/extensions/queue/queue.ts

```diff
@@ -0,0 +1,389 @@
+import * as path from 'path';
```

Comment:
dumb question: do cdk source files need to have a copyright header?

Reply:
Our modules have a [NOTICE file](https://github.com/aws/aws-cdk/blob/master/packages/%40aws-cdk-containers/ecs-service-extensions/NOTICE) so we're good 😄